feather/worldgen/src/lib.rs

```diff
@@ -364,6 +364,18 @@ impl BiomeGenerator for StaticBiomeGenerator {
     }
 }
 
+pub fn seed_from_string(s: &str) -> u64 {
```

Comment:
This should maybe be an i64, such that negative seeds also work as expected?

Reply:
Everywhere in feather seeds are defined by u64

Reply:
Yes, but it should be parsed as `i64` and then cast to `u64`.

Reply:
I think it should be `i64` everywhere and only be casted when passing it to RNGs, it would be more convenient for feather or plugin developers who expect `-104019040` seed to be negative, and for end users that expect /seed (or seed value in logs / crash reports) to be the same as in config.toml.

Reply:
I agree with the above comment, @ELginas do you want to make this change or should I merge this? We can then create a new issue to change the seed type from `u64` to `i64`.